dmoj/checkers/_checker.c

```diff
@@ -10,6 +10,16 @@
 #	define inline
 #endif
 
+#if PY_MAJOR_VERSION >= 3
```

Comment:
We should shim python 3 functions on python 2 instead. No point in being fair to python 2.

Reply:
That feels like it could be confusing &mdash; maybe not here, but if this were bigger than definitely confusing.

Reply:
It's fairly standard practice since dropping python 2 support will just be removing the macros instead of changing all the references.